docs/source/executing-operations/defer-support.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Apollo Router support for @defer
+description: Incremental delivery of entity fields
+---
+
+> This feature is currently in **preview**, available in Apollo Router v1.0 and later.
+
+Queries sent to the Apollo Router can use the `@defer` directive to enable the incremental delivery of data and improve the router's initial response time. This functionality is contained entirely within the router and does _not_ require your subgraphs to also support `@defer`!
+
+## What is `@defer`?
+
+The `@defer` directive ([see the RFC](https://github.com/graphql/graphql-wg/blob/main/rfcs/DeferStream.md)) enables a client query to specify sets of fields that it doesn't need to receive data for _immediately_. This is helpful whenever some fields in a query take much longer to resolve than others.
+
+Here's an example query that uses `@defer`:
+
+```graphql
+query GetPersonAndFriends($personId: ID!) {
+  person(id: $personId) {
+    # Basic fields (fast)
+    id
+    name {
+      firstName
+      lastName
+    }
+
+    # highlight-start
+    # Friend fields (slower)
+    ... on User @defer {
+      friends {
+        id
+      }
+    }
+    # highlight-end
+  }
+}
+```
+
+When responding to the above query, a GraphQL server can "chunk" its response, starting by returning only the "basic fields". Later, it can return an additional chunk that includes the "friend fields".
```

Comment:
maybe another term than `chunk` would be better here, as it has a very specific meaning for people manipulating HTTP, and that would create invalid assumptions on the underlying transport (deferred responses are transported as HTTP body chunks, but there's no guarantee that a proxy would not cut or reassemble chunks)

Reply:
what clients have to support is multipart encoded responses, where each deferred response is a part

Reply:
agree that chunk might be an implementation detail that could vary.  just bikeshedding:

* "can return its response gradually"
* "can return an initial response as soon as possible and send additional follow-up responses as they become available"

Reply:
I have "de-chunkified" the article 👍